drivers/Watchdog.h

```diff
@@ -44,62 +47,67 @@ namespace mbed {
  * @endcode
  * @ingroup drivers
  */
-class Watchdog {
+class Watchdog
+{
 public:
     Watchdog() {}
```

Comment:
There's a bunch of uninitialised members if we use this one

Reply:
Also the compiler won't know when to call this as you have default argument values in the other constructor